tick/u.q

```diff
@@ -3,16 +3,16 @@
 /2006.05.08 add
 
 \d .u
-init:{w::t!(count t::tables`.)#()}
+init:{(w;t)::(t!();tables`.)}
 
-del:{w[x]_:w[x;;0]?y};.z.pc:{del[;x]each t};
+(del;.z.pc):({w[x]_:w[x;;0]?y};{del[;x]each t})
```

Comment:
There isn't really a point on doing this, since the original code should have been switched to two lines instead in the first place:

```q
del:{w[x]_:w[x;;0]?y};
.z.pc:{del[;x]each t};
```

It's a cool use of the new features v4.1 offers, but I don't really see the point.

Reply:
Yes, as I said before I tried to incorporate as many changes as possible, but in this case I think this doesn't makes sense.